content/articles/2018-08-05-google-summer-of-code-2018.md

```diff
@@ -0,0 +1,24 @@
+Title: GSoC'18: improving propagators, adding natural and artificial perturbations at Poliastro
+Date: 2018-08-05 14:00
+Category: GSoC
+Tags: GSoC, poliastro
+slug: 2018-08-05-google-summer-of-code-2018
+lang: en
+Author: Nikita Astrakhantsev
+
+It was a lazy sunday at home and nothing prepared me for my GF giving me the link for the GSoC'18 announcement page. I was considering participation a year ago, but had not enough resoluteness for that. This year I wanted it to be different so I started scrolling through the list of open-source organisations. As I am doing a PhD in theoretical physics now (defending in Jnauary 2019), I wanted the project to be scientific, not just some boring back-end development (in which I am anyway not strong enough). After a couple of iterations I converged to three space-aimed organisations. What shocked me, was the fact that mostly people of these organizations were not eager to even answer simple questions or it took them a week to provide a dataset for analysis. Only in Poliastro it was easy enough to get in touch with people and they (especially Juan) turned out to be friendly and helpful. In such an atmosphere it is more than possible to work.
```

Comment:
~~sunday~~ Sunday, ~~Jnauary~~ January, ~~Juan~~ Juan Luis (that's my full name :innocent:)

Reply:
what's "my GF"? possibly I'm missing some silly thing here :)

Reply:
> I wanted the project to be scientific, not just some boring back-end development

haha, I feel your pain!!